packages/core/src/model.ts

```diff
@@ -26,7 +26,8 @@ export function createModel<
   >,
   TComputedAction = UnionFromCreatorsReturnTypes<
     Prop<TFinalModelCreators, 'actions'>
-  >
+  >,
+  TComputedInput = Prop<TFinalModelCreators, 'input'>
```

Comment:
This is unused and probably should stay that way as `withInput` already is responsible for accepting this
```suggestion
  >
```